### PR TITLE
fix json_rewrite_test CRLF handling

### PR DESCRIPTION
**Pull request by jtattermusch**
fixes json_rewrite_test on windows on jenkins. Part of #1912.

### CODE REVIEW

File: test/core/json/json_rewrite_test.c [hunk 1]
```diff
@@ -64,6 +64,11 @@ typedef struct json_reader_userdata {
 static void json_writer_output_char(void* userdata, char c) {
   json_writer_userdata* state = userdata;
   int cmp = fgetc(state->cmp);
+
+  /* treat CRLF as LF */
+  if (cmp == '\r' && c == '\n') {
+	  cmp = fgetc(state->cmp);
```
Comment (nicolasnoble) [inline test/core/json/json_rewrite_test.c]: There's a bad indentation here :-)

**Comment by jtattermusch** [comment]
Indentation fixed.

**Comment by nicolasnoble** [comment]
retest this please

**Comment by nicolasnoble** [comment]
test this please

**Comment by jtattermusch** [comment]
test this please

**Comment by jtattermusch** [comment]
test this please

**Comment by ctiller** [comment]
test this please (this is fun)

**Comment by jtattermusch** [comment]
test this please

**Comment by nicolasnoble** [comment]
test this please

**Comment by jtattermusch** [comment]
how about we merge this one and start messing with some newer pull request?

**Pull request merged**